docs/guides/admin/job_queue_flow.md

```diff
@@ -1,5 +1,7 @@
 ![SeAT](http://i.imgur.com/aPPOxSK.png)
 
+# Job Queue Flow
+
 The current flow of things when a job enters the system can be seen below:
 
 ![Flow](http://i.imgur.com/TPgha3P.png)
```

Comment:
Image need an Update.
HighQueue contains Key as name

Reply:
@leonjza go here

Reply:
wrt to docker ....

Docker cannot be used with VMware on the same windows installation. ( ie it cannot even be INSTALLED on the same machine as VMware ) so for anyone developing on a Windows environment ( yes I know its not "supported" ) docker may be a non starter. It is important to have the OPTION to not use docker.

Reply:
For developing with Windows+VMware SeAT Supports a Vagrant Machine: https://github.com/eveseat/scripts/pull/17/commits/81b5468c2ba199b84c1b266329b19be28f6349bc 

However the OPTION is still there.